.github/workflows/ci-build.yml

```
@@ -172,4 +172,53 @@ jobs:
         if: always()
         with:
           name: ledfx-log-${{ runner.os }}-${{ runner.arch }}-py${{ matrix.python-version }}.log
-          path: ${{ github.workspace }}/debug_config/ledfx.log
\ No newline at end of file
+          path: ${{ github.workspace }}/debug_config/ledfx.log
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)